bootstrap.sh

```diff
@@ -81,6 +81,22 @@ fi
 [ -f "${SCRIPTS_DIR}/config.sh" ] && . "${SCRIPTS_DIR}/config.sh"
 [ -f "${SCRIPTS_DIR}/config_${SERVER_ENV}.sh" ] && . "${SCRIPTS_DIR}/config_${SERVER_ENV}.sh"
 
+# Check for sync folder, and make sure it's empty
+if [[ -d "../${HYDRA_HEAD}/" && $(ls -A "../${HYDRA_HEAD}/") ]]; then
+    echo "The local directory '../${HYDRA_HEAD}/' exists, and is not empty."
+    echo "If you continue, this directory will be deleted. You may lose your work."
+    # Only accept an answer for five seconds.
+    read -t 5 -p 'Do you want to continue? [y|N] '
+    # Only proceed if something was read, and it was some form of yes.
+    if [ $? -eq 0 && $REPLY =~ ^[Yy]([Ee][Ss])?$ ]]; then
```

Comment:
I am getting this error when testing:

./bootstrap.sh: line 91: syntax error near unexpected token `('


So, either REPLY is not being set or else the regex needs to be quoted?